### PR TITLE
chore: simplify build and cleanup leftovers

### PR DESCRIPTION
**Pull request by suevalov**
# Purpose of PR

* Simplifies build in the umbrella package - uses parcel instead of rollup.
* Removes Dropdown component
* Removes fcss package

### CODE REVIEW

**Comment by denkristoffer** [comment]
> - Removes fcss package

Awesome! that also means we can build with node 16 going forward 🎉🥳

**Comment by burakukula** [comment]
Cool!
Tbh I was not aware that we want to remove `fcss` package in v4 (or I don't remember this discussion 😬). Do we have any depreciation plan for it? I think we don't use it, but maybe someone else does?

File: packages/forma-36-react-components/package.json [hunk 1]
```diff
@@ -2,25 +2,19 @@
...
+  "sideEffects": false,
+  "browserslist": "extends @contentful/browserslist-config",
   "scripts": {
-    "build:clean": "rm -rf dist esm",
-    "build:scripts": "BABEL_ENV=build yarn babel src --out-dir dist --ignore 'src/**/*.stories.js','src/**/*.stories.tsx','src/**/*.test.tsx','src/**/*.test.js','src/**/*.css.d.ts' --extensions '.ts','.tsx','.js'",
-    "build:esm": "BABEL_ENV=esm yarn babel src --out-dir esm --ignore 'src/**/*.stories.js','src/**/*.stories.tsx','src/**/*.test.tsx','src/**/*.test.js','src/**/*.css.d.ts' --extensions '.ts','.tsx','.js' && rollup -c tools/rollup.esm.config.js && rm -rf esm",
-    "build:styles": "yarn webpack --config ./tools/webpack.production.config.js",
-    "build:types": "tsc --emitDeclarationOnly --noEmit false --declaration true",
-    "build:post-clean": "rm -rf dist/styles.js dist/main.js",
-    "build": "yarn build:clean && yarn build:scripts && yarn build:esm && yarn build:styles && yarn build:types && yarn build:post-clean && yarn size",
```
Comment (gui-santos) [inline packages/forma-36-react-components/package.json:23]: Beautiful

**Review by gui-santos — APPROVED** [review]
💎 

Amazing

**Comment by gui-santos** [comment]
> Tbh I was not aware that we want to remove `fcss` package in v4 (or I don't remember this discussion 😬). Do we have any depreciation plan for it? I think we don't use it, but maybe someone else does?

@burakukula I think we can tell people to use `Box` and our tokens like @suevalov did in some places of the code
they can also stay in v3 with fcss if they want

**Comment by suevalov** [comment]
> Cool!
> Tbh I was not aware that we want to remove `fcss` package in v4 (or I don't remember this discussion 😬). Do we have any depreciation plan for it? I think we don't use it, but maybe someone else does?

I think it was a plan from the beginning (at least it was my understanding). I guess people who are using fcss can continue using v3 if they want.

**Comment by suevalov** [comment]
https://github.com/contentful/forma-36/pull/1351/files#diff-35020f1a5865f9eb95e12f3a5921c2e65728323b8dbde43993e0c561c077ac1eR6

I've managed to shave 200 Kb off in the V4 (even without tree-shaking)

**Review by denkristoffer — APPROVED** [review]

**Review by burakukula — APPROVED** [review]

**Pull request merged**